definitions/aircraft/Asobo_KingAir.yaml

```diff
@@ -245,4 +251,20 @@ shared:
     var_type: f64
     event_name: LIGHT_POTENTIOMETER_SET
     event_param: 14
-    index_reversed: true
\ No newline at end of file
+    index_reversed: true
+  - # Pressurization
+    type: NumIncrement
+    var_name: A:PRESSURIZATION CABIN ALTITUDE GOAL
+    var_units: Feet
+    var_type: i32
+    up_event_name: PRESSURIZATION_PRESSURE_ALT_INC
+    down_event_name: PRESSURIZATION_PRESSURE_ALT_DEC
+    increment_by: 63
+  -
+    type: NumIncrement
+    var_name: A:PRESSURIZATION CABIN ALTITUDE RATE
+    var_units: Feet
+    var_type: i32
+    up_event_name: PRESSURIZATION_CLIMB_RATE_INC
+    down_event_name: PRESSURIZATION_CLIMB_RATE_DEC
+    increment_by: 63
```

Comment:
These could probably go under pressurization.yaml for other aircraft to use. That said, these things I feel are best kept as vars for a universal basis and incrementing might just cause inaccuracy.